### PR TITLE
Add localization support to music and video sensing extensions

### PR DESCRIPTION
**Pull request by thisandagain**
### Proposed Changes

- Adds localization support to the music extension
- Adds localization support to the video motion extension
- Combines all extension strings during export into a single `en.json` file as per discussion with @chrisgarrity

### CODE REVIEW

**Comment by thisandagain** [comment]
Example of output after running `npm run i18n:src`:
[en.json.zip](https://github.com/LLK/scratch-vm/files/1992698/en.json.zip)

File: src/extensions/scratch3_music/index.js [hunk 1]
```diff
@@ -192,75 +193,147 @@ class Scratch3MusicBlocks {
...
+                    description: 'Sound of two hands clapping together'
+                }),
                 fileName: '8-hand-clap'
             },
             {
-                name: '(9) Claves',
+                name: formatMessage({
+                    id: 'music.drumClaves',
+                    default: '(9) Claves',
+                    description: 'Sound of a claves being struck together'
```
Comment (ericrosenbaum) [inline src/extensions/scratch3_music/index.js]: Remove the 'a'

File: src/extensions/scratch3_video_sensing/index.js [hunk 2]
```diff
@@ -318,15 +335,27 @@ class Scratch3VideoSensingBlocks {
...
+                    description: 'Option for the "turn video [STATE]" block'
+                }),
                 value: VideoState.ON
             },
             {
-                name: 'on flipped',
+                name: formatMessage({
+                    id: 'videoSensing.onFlipped',
+                    default: 'on flipped',
+                    description: 'Option for the "turn video [STATE]" block'
```
Comment (ericrosenbaum) [inline src/extensions/scratch3_video_sensing/index.js]: Might help to add some clarification for translators of what we mean by "flipped". Maybe something like: 

'Option for the "turn video [STATE]" block that causes the video to be flipped horizontally (reversed as in a mirror)'

File: src/extensions/scratch3_video_sensing/index.js [hunk 3]
```diff
@@ -376,7 +413,11 @@ class Scratch3VideoSensingBlocks {
                 },
                 {
                     opcode: 'videoToggle',
-                    text: 'turn video [VIDEO_STATE]',
+                    text: formatMessage({
+                        id: 'videoSensing.videoToggle',
+                        default: 'turn video [VIDEO_STATE]',
+                        description: 'Controls dispay of the video preview layer'
```
Comment (ericrosenbaum) [inline src/extensions/scratch3_video_sensing/index.js]: typo

**Review by ericrosenbaum — APPROVED** [review]
Looks good, pending a couple typos (and a suggestion)

**Pull request merged**